scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityExpression.java

```diff
@@ -77,13 +77,13 @@ protected ActivityExpression() { }
     Map<String, SerializedValue> arguments = new HashMap<>();
     Map<String, VariableArgumentComputer> variableArguments = new HashMap<>();
 
-    public B withArgument(String argument, QueriableState state, TimeExpression timeToQuery) {
-      variableArguments.put(argument, new StateQueryParam(state, timeToQuery));
+    public B withArgument(String argument, String nameState, TimeExpression timeToQuery) {
```

Comment:
[minor] same as the `nameResource` comment, calling it `nameState` sounds like a variable representing "the state of the name" rather than "name of the state"